contrib/awssqs/samples/README.md

```diff
@@ -0,0 +1,117 @@
+# AWS SQS - Source
+
+These samples demonstrate how to configure an AWS SQS source, consuming
+events from an AWS SQS queue.
+
+## Deployment Steps
+
+The steps below assume that you have cloned the
+[eventing-sources](https://github.com/knative/eventing-sources) repo and
+run the commands from contrib/awssqs.
+
+### Prerequisites
+
+1.  Create an [AWS SQS queue](https://aws.amazon.com/sqs/).
+
+1.  Setup
+    [Knative Eventing](https://github.com/knative/docs/tree/master/eventing).
+
+1.  The [in-memory `ClusterChannelProvisioner`](https://github.com/knative/eventing/tree/master/config/provisioners/in-memory-channel)
+    should be installed in your cluster. At the time of writing (release
+    v0.4.0) it is part of the default instructions so you will probably
+    have it there already.
+
+### Create a channel and subscriber
+
+```shell
+ko apply -f samples/display-resources.yaml
+```
+
+The sample provided will configure an in-memory channel (named
+`awssqs-test` and a subscription to a consumer named `event-display`)
+that will simply display incoming events to stdout.
+
+At this point we have a channel and a subscriber ready to receive
+events, we will now create an awssqs source resource that starts
+pulling events from a SQS queue.
+
+### Set up credentials
+
+Acquire [AWS Credentials](https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html)
+for the same account. Your credentials file should look like this:
+
+     [default]
+     aws_access_key_id = ...
+     aws_secret_access_key = ...
+
+Then create a secret for the downloaded key:
+
+ ```shell
+ kubectl -n knative-sources create secret generic awssqs-source-credentials --from-file=credentials=PATH_TO_CREDENTIALS_FILE
+ ```
+
+### Deployment
+
+Deploy the `AwsSqsSource` controller as part of eventing-source's controller.
+
+   ```shell
+   ko -n default apply -f config/
```

Comment:
This is wrong. It should be `ko -n default apply -f contrib/awssqs/`

Reply:
It should be correct but it's relative to contrib/awssqs. The [Deploy section above](https://github.com/knative/eventing-sources/pull/253/files#diff-38fc07ee88fe2e61c7a0f8c6c3e62604R10) indicates that commands are run from that dir.

I understand however that you prefer to have the commands relative to the root of the repo (that is the case for instructions in other sources in contrib/). I'll change it.

Reply:
Done.